src/toast/todmap/pointing.py

```diff
@@ -165,8 +165,6 @@ def exec(self, data):
                 hwpang = tod.local_hwp_angle()
             except:
                 pass
```

Comment:
Seems like assigning `hwpang = None` in the except clause would be prudent. I have been burned by overly general try-catch segments before. Perhaps it would make sense to list the exceptions we anticipate?

Reply:
Going recursively through the code, it looks like the call to `local_hwp_angle()` can raise `RuntimeError`, `ValueError`, and `NotImplementedError`.  I guess we need a cleaner way to determine if a TOD instance has HWP angles...